src/include/migraphx/op/concat.hpp

```diff
@@ -73,49 +74,87 @@ struct concat
         }
         return offsets;
     }
+
     shape normalize_compute_shape(std::vector<shape> inputs) const
     {
-        if(inputs.empty())
+        // inputs can contain 1 or more shapes (variadic).  compute_shape_op ensures there must
+        // be at least 1.
```

Comment:
update comment

Reply:
That ***is*** an update.  That check is performed in operation.hpp:143 and means that if we get to this point in the code, there's nothing to check for.